src/uv-common.c

```diff
@@ -342,7 +342,7 @@ int uv_tcp_bind(uv_tcp_t* handle,
 }
 
 
-int uv_udp_init_ex(uv_loop_t* loop, uv_udp_t* handle, unsigned flags) {
+int uv_udp_init_ex(uv_loop_t* loop, uv_udp_t* handle, unsigned int flags) {
```

Comment:
If VS struggles with this, then... it's simply not very good. Writing `unsigned` instead of `unsigned int` is pretty prevalent in libuv, this is just one instance.

Colin already signed off on this so I'm not going to block merging it but IMO you're better off reporting this to VS and have them fix their analyzer. Address the cause, not the symptom, so to speak.

Reply:
> 
For a type to be automatically recognized by the compiler, I think a warning from Microsoft is not unreasonable. In a standard type definition, we need to specify its type, not rely on the compiler to automatically identify it.

Reply:
Let me clarify: `unsigned x` and `unsigned int x` mean exactly the same thing, they're semantically equivalent. A compiler will treat them as such (because it's per spec) and so any source code analyzer worth its salt should too.

Reply:
I also think @bnoordhuis is right on this, this needs to be reported to Microsoft.